nx.json

```diff
@@ -0,0 +1,20 @@
+{
+  "extends": "nx/presets/npm.json",
+  "tasksRunnerOptions": {
+    "default": {
+      "runner": "nx/tasks-runners/default",
+      "options": {
+        "cacheableOperations": ["build", "example", "test", "lint", "docusaurus"],
+        "accessToken": "NWY1YmQ1MDctYzQ0Ny00OTAzLTgzYWUtZWQwZmY2ODRlZWFlfHJlYWQtd3JpdGU="
```

Comment:
Let's not publish tokens on GH please. This token is considered as leaked and should be rotated.

We should use `NX_CLOUD_ACCESS_TOKEN` env variable as pointed out in https://nx.dev/nx-cloud/account/access-tokens#setting-a-different-access-token-in-ci

Reply:
As dicussed, we are not using nx-cloud, so the `accessToken` property is redundant. Still, I agree it should be removed, to avoid confusion.

Reply:
Notice that we specify `nx/tasks-runners/default` runner: this does not employ the [nx-cloud runner](https://cloud.nx.app/orgs/workspace-setup).